spec.bs

```diff
@@ -4275,6 +4351,9 @@ An <dfn>interest group</dfn> is a [=struct=] with the following [=struct/items=]
     </p>
   : <dfn>trusted bidding signals keys</dfn>
   :: Null or a [=list=] of [=string=]. See [=building trusted bidding signals url=].
+  : <dfn>trusted bidding signals slot size mode</dfn>
+  :: "`none`", "`slot-size`" or "`all-slots-requested-sizes`". Defaults to "`none`".

```

Comment:
Nit: “Defaults to” -> "Initially". That's the term I was suggested to use.

Reply:
Done.